security/application-roles.md

```diff
@@ -1,4 +1,22 @@
 ---
 description: >-
   Describes the application roles used by SysKit Sense, what they mean and how to set them up.
----
\ No newline at end of file
+---
+# Application roles in SysKit Sense
+
+SysKit Sense have integrated Role-Based access control (RBAC). It leverages the Application Roles feature of Azure Active Directory Applications. SysKit Sense ships with the following Application Roles:
+- **Viewer** - Reserved for future usage

```

Comment:
I actually removed this when i changes contributor to admin

Reply:
so no viewer any more?

Reply:
correct